numba/tests/test_return_values.py

```diff
@@ -61,13 +58,12 @@ def test_pyobj_func_npm(self):
     def test_module_func(self, flags=enable_pyobj_flags):
         # Test returning imported func that is only supported in object mode
         pyfunc = get_module_func
-        cr = compile_isolated(pyfunc, (), flags=flags)
-        cfunc = cr.entry_point
+        cfunc = jit((), **flags)(pyfunc)
         if flags == enable_pyobj_flags:
             result = cfunc()
             self.assertEqual(result, math.floor)
         else:
-            result = cfunc()
+            cfunc()
```

Comment:
Dead code because the compiler would have failed.

Reply:
Thanks, fixed in 90ffc0544d1.